README.md

```diff
@@ -65,6 +65,11 @@
  ```
   $ cp .env-example .env
  ```
+ - Installing Postgres
+```
```

Comment:
On the README, set PostgreSQL as a preriquisite because we can install PostgreSQL on many different machines using many different methods, not just using `apt-get`.